widget/popup_test.go

```diff
@@ -20,6 +21,80 @@ func TestNewPopUp(t *testing.T) {
 	assert.Equal(t, pop, test.Canvas().Overlays().List()[0])
 }
 
+func TestShowPopUp(t *testing.T) {
+	require.Nil(t, test.Canvas().Overlays().Top())
```

Comment:
test.Canvas() is called multiple times in this test, does it change or can it be a variable?

Reply:
It's immutable, I think. But honestly, I don't think it is of much importance in tests if it does not affect the test's comprehensibility.

Reply:
Fair enough